Script/char.dat

```diff
@@ -802,6 +840,61 @@ guard
       "\"Not that I think about it, I was reassigned as an honor guard after that huge screw-up. I wonder if... Nah, it's probably just a coincidence.\"";
     }
   }
+
+  Config EMISSARY;
```

Comment:
I think this guy needs a complete rewrite; he doesn't subvert, parody or criticise anything, his "need help, civil war" schtick is very standard-fantasy, and his country's enslavement of goblins undermines IVAN's criticism of slavery (such as it is) because no-one else has any lines talking about it and none of the goblins in the game (even Guugzamesh) have any interesting lines.